product_docs/docs/pgd/5/rel_notes/pgd_5.5.0_rel_notes.mdx

```diff
@@ -0,0 +1,86 @@
+---
+title: "EDB Postgres Distributed 5.5.0 release notes"
+navTitle: "Version 5.5.0"
+---
+
+Released: 14th May 2024
+
+EDB Postgres Distributed version 5.5.0 is a minor version of EDB Postgres Distributed.
+
+!!! Important Recommended upgrade
+We recommend that all users of PGD 5 upgrade to PGD 5.5. See [PGD/TPA upgrades](../upgrades/tpa_overview) for details.
+!!!
+
+
+## Highlights of EDB Postgres Distributed 5.5.0
+
+Highlights of this 5.5.0 release include:
+
+* read scalability enhancements in PGD Proxy which allow read-only queries to be routed to nodes that are not the write leader, improving the overall performance of the cluster.
+
+## Compatibility
+
+!!! Note EDB Server version compatibility
+This version requires the recently released Postgres versions 14.10, 15.4,
+or 16.1 (or later) of EDB Postgres Advanced Server or EDB Postgres Extended
+Server. No such restrictions exist for PostgreSQL Server.
```

Comment:
```suggestion
Server. No such restrictions exist for the PostgreSQL server.
```
?? I don't see a product called PostgreSQL Server, so I think this is what was meant?